layouts/partials/security-platform/appsec-languages.html

```diff
@@ -52,10 +52,10 @@
         </a>
       </div>
       <div class="col">
-        <a class="card h-100" href="/security_platform/application_security/how-appsec-works/#serverless-monitoring">
+        <a class="card h-100" href="serverless">
           <div class="card-body text-center py-2 px-1">
-          <strong>BETA</strong><br/>
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/amazon_lambda.png" "class" "img-fluid" "alt" "AWS Lambda" "width" "400") }}
+          BETA: Serverless<br/>
+            {{ partial "img.html" (dict "root" . "src" "security/application_security/serverless.png" "class" "img-fluid" "alt" "Serverless" "width" "400") }}
```

Comment:
small nit, is it possible to reduce the picture size here? since the image resolution is low, it comes up a bit grainy